command/healthcheck/pki_role_allows_glob_wildcards.go

```diff
@@ -49,7 +49,7 @@ func (h *RoleAllowsGlobWildcards) FetchResources(e *Executor) error {
 	})
 	if exit || err != nil {
 		if f != nil && f.IsSecretPermissionsError() {
-			h.NoPerms = true
+			h.FetchIssue = f
```

Comment:
I actually kinda like `hardware_backed_root`'s approach of having a map here tbh.

Reply:
Did both, kept the FetchIssue for failure to list roles which becomes a complete failure for the role check, otherwise we do like the hardware_backend_root and fail just the specific role